src/debug/debugSession.ts

```diff
@@ -49,7 +52,10 @@ export class DebugSession implements IDebugSession {
         // TODO: Support docker-compose.yml
         const dockerfilePath = path.join(workspaceFolder.uri.fsPath, "Dockerfile");
         if (!fs.existsSync(dockerfilePath)) {
-            vscode.window.showErrorMessage(`No Dockerfile found in the workspace ${workspaceFolder.name}`);
+            const answer = await vscode.window.showErrorMessage(`No Dockerfile found in the workspace ${workspaceFolder.name}, please refer to the wiki for the usage.`, "Open in Browser");

```

Comment:
Avoid run-on sentence, and don't refer to the article as a 'wiki'. E.g. `No Dockerfile found in the workspace ${…}. See the documentation for how to use this command.`

Reply:
Thanks, Nice suggestion. Learned it.